pytest_structlog.py

```diff
@@ -37,6 +39,16 @@ def __lt__(self, other):
 absent = object()
 
 
+def level_to_name(level):
+    """Given the name or number for a log-level, return the lower-case level name."""
+    if isinstance(level, str):
+        return level.lower()
+    try:
+        return logging._levelToName[level].lower()
```

Comment:
It looks like [`logging.getLevelName`](https://docs.python.org/3/library/logging.html#logging.getLevelName) is the public API here.

Reply:
Ah, I wasn't aware of that!

I also seem to have broken the tests in 2.7 for no good reason.

Reply:
That function seems to be a minor shitshow but it should be safe to pass integers into it.

> Changed in version 3.4: In Python versions earlier than 3.4, this function could also be passed a text level, and would return the corresponding numeric value of the level. This undocumented behaviour was considered a mistake, and was removed in Python 3.4, but reinstated in 3.4.2 due to retain backward compatibility.